### ISSUE TITLE
CentOS 8 template

### ISSUE THREAD

**Issue by fepitre** [QubesOS/qubes-issues#5755]
Tasks:
- [x] build all packages
- [x] build the template
- [x] document
- [x] upload to testing repo
- [x] migrate to stable repo
- [x] announce

If any issue affects CentOS 8 specifically (build failures, things that worked fine before etc), please add reference to this issue too.

cc @marmarek

**Comment by fepitre** [comment]
For you information, the template is working. Several commits have been merged in the past months and right now, we are using Fedora COPR for improving missing packages for CentOS7+: https://copr.fedorainfracloud.org/coprs/fepitre/epel-8-qubes/

**Comment by xaki23** [comment]
currently seeing two issues with the c8 202010191306 template in community-testing:

some kind of deps-chain issue:
```
Error: 
 Problem: cannot install both libxfce4ui-4.16.0-2.el8.x86_64 and libxfce4ui-4.14.1-3.el8.x86_64
  - package garcon-1000:0.6.4-4.centos8.x86_64 requires libxfce4ui-1.so.0()(64bit), but none of the providers can be installed
  - cannot install the best update candidate for package libxfce4ui-4.14.1-3.el8.x86_64
  - cannot install the best update candidate for package garcon-1000:0.6.4-4.centos8.x86_64
(try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

and after installing the other ca 9348756 updates with --nobest (to avoid removing some qubes-pkg), there is an issue with the repo config in general:
```
Errors during downloading metadata for repository 'qubes-vm-r4.1-current':
  - Status code: 404 for https://yum.qubes-os.org/r4.1/current/vm/8/repodata/repomd.xml (IP: 127.0.0.1)
Error: Failed to download metadata for repo 'qubes-vm-r4.1-current': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```
note the part with the /8/ instead of /centos8/, so the "centos" prefix got dropped from '$releasever"

**Comment by Osndok** [comment]
I ran into the same issues. The first error is odd. The qubes repo pulls in an order version using a large epoch, which would indicate that it is trying to mask the `garcon` that is coming from `epel`... but... I don't see why. There is not a comment in the overriding package description, and trying to use the old version looks like it breaks a bunch of other `epel` deps that the template wants.

Adding `exclude = garcon` to all the repos in the qubes repo config lets the template `distrosync`, so this seems to be the only packaging issue.

**Does anyone have more info on why we might need an old version of garcon?**

I was able to work around the second error by doing:
```
sed -i.orig -e 's|vm/$rel|vm/centos$rel|g' qubes-r4.repo
```

I don't know what causes this, but I did (at least once) get an error like this from the template. Maybe this is the consequence for not having the older `garcon` binaries?
```
failed to execute qubes.GetDate from rocky-8
```
...we'll see if it continues.
